ui/src/project.ts

```diff
@@ -94,12 +188,37 @@ interface FetchList extends event.Event<Kind> {
   urn?: string;
 }
 
+interface FetchPeers extends event.Event<Kind> {
+  kind: Kind.FetchPeers;
+  urn: urn.Urn;
+}
+
 interface FetchLocalState extends event.Event<Kind> {
   kind: Kind.FetchLocalState;
   path: string;
 }
 
-type Msg = ClearLocalState | Create | Fetch | FetchList | FetchLocalState;
+interface TrackPeer extends event.Event<Kind> {
```

Comment:
Should this be `FollowPeer`? :eyes:

Reply:
Only if we go all the way and also rename the proxy endpoints. Not sure it's worth the effort at the moment.